pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -147,6 +152,9 @@ type VolumeToAttach struct {
 	// NodeName is the identifier for the node that the volume should be
 	// attached to.
 	NodeName string
+
+	// Pod to mount the volume to. Used to create NewMounter.
+	Pod *api.Pod
```

Comment:
@saad-ali  - sorry for my newbie questions, but I'm having trouble finding how I would pass a ref from a runtime pod object in this case.  I see all through the code that *api.Pod is being passed a whole bunch, but yet to find a way or example of how to pass the ref?  Can you point me to an example of how I would reference a runtime pod object to pass into api.GetReference, I'm not finding much info on referencing objects in golang?  Or maybe I'm not understanding what you mean?